paasta_tools/marathon_tools.py

```diff
@@ -326,8 +333,21 @@ def format_marathon_app_dict(self, app_id, docker_url, docker_volumes, service_n
                                marathon configuration file
         :param service_namespace_config: The service instance's configuration dict
         :returns: A dict containing all of the keys listed above"""
+        instances = self.get_instances()
+
+        CONFIG_HASH_BLACKLIST = set(['instances', 'backoff_seconds'])
+
+        partial_id = format_job_id(service=self.service, instance=self.instance)
+        system_paasta_config = load_system_paasta_config()
+        docker_url = get_docker_url(system_paasta_config.get_docker_registry(), self.get_docker_image())
+        service_namespace_config = load_service_namespace_config(
+            service=self.service,
+            namespace=self.get_nerve_namespace(),
+        )
+        docker_volumes = system_paasta_config.get_volumes() + self.get_extra_volumes()
+
         complete_config = {
-            'id': app_id,
+            'id': partial_id,
```

Comment:
Just junk this and the whole concept of a partial id

I know it will cause a bounce, this is fine